examples/plot_simulate_alpha.py

```diff
@@ -54,18 +53,42 @@
 dpl = simulate_dipole(net, postproc=False)
 
 ###############################################################################
-# We can confirm that what we simulate is indeed 10 Hz activity by plotting the
-# power spectral density (PSD). Note that the SciPy-function
-# `~scipy.signal.spectrogram` is used to create the plot. The
-# ``dpl[trial_idx].scale()`` call relates to the amount of cortical tissue
-# necessary to observe the electric current dipole outside the head with M/EEG.
+# Prior to plotting, we can choose to smooth the dipole waveform (note that the
+# :meth:`~hnn_core.dipole.smooth`-method operates in-place, *i.e.*, it alters
+# the data inside the ``Dipole`` object). Smoothing approximates the effect of
+# signal summation from a larger number and greater volume of neurons than are
+# included in our biophysical model. Note also that the native units of the
+# simulated dipole is nAm. We can confirm that what we simulate is indeed
+# 10 Hz activity by plotting the power spectral density (PSD).
 import matplotlib.pyplot as plt
-from hnn_core.viz import plot_dipole, plot_spectrogram
-trial_idx = 0  # single trial simulated
+from hnn_core.viz import plot_dipole, plot_psd
+
 fig, axes = plt.subplots(2, 1)
-tmin = 20  # exclude initial burn-in period
-plot_dipole(dpl[trial_idx], tmin=tmin, ax=axes[0], show=False)
-plot_spectrogram(dpl[trial_idx], fmin=0., fmax=40., tmin=tmin, ax=axes[1])
+tmin, tmax = 10, 300  # exclude the initial burn-in period from the plots
+trial_idx = 0  # single trial simulated
+
+dpl[trial_idx].convert_fAm_to_nAm()
```

Comment:
data should always be in nAm. I guess Neuron spits out fAm? Let's convert it to nAm and then user doesn't have to know about these conversions.

Reply:
Yes, this was my proposal.